lib/src/main/java/com/github/omadahealth/lollipin/lib/managers/AppLockImpl.java

```diff
@@ -371,6 +376,24 @@ public boolean shouldLockSceen(Activity activity) {
         return true;
     }
 
+    @Override
+    public int getDefaultMaxLoginAttemptsAllowed() {
+        //default number of attempts allowed
+        return AppLock.DEFAULT_MAX_LOGIN_ATTEMPTS_ALLOWED;
+    }
+
+    @Override
+    public void resetLoginAttempts() {
+        mSharedPreferences.edit().putInt(NUM_OF_ATTEMPTS, 0).apply();
+    }
+
+    @Override
+    public int getLoginAttempts() {
```

Comment:
Maybe call this `incrementAndGetLoginAttempts()` to make it clear it has side effects.